src/current/_includes/releases/v23.1/v23.1.25.md

```diff
@@ -0,0 +1,80 @@
+## v23.1.25
+
+Release Date: August 15, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-25-security-updates">Security updates</h3>
+
+- Secrets are now sanitized from URLs before being written to unredacted [logs]({% link v23.1/configure-logs.md %}) when used in the following SQL statements:
+    - [`BACKUP`]({% link v23.1/backup.md %})
+    - [`RESTORE`]({% link v23.1/restore.md %})
+    - [`CREATE SCHEDULE FOR BACKUP`]({% link v23.1/create-schedule-for-backup.md %})
+    - [`ALTER BACKUP SCHEDULE`]({% link v23.1/alter-backup-schedule.md %})
+    - [`ALTER BACKUP`]({% link v23.1/alter-backup.md %})
+    - [`SHOW BACKUP`]({% link v23.1/show-backup.md %})
+    - [`CREATE CHANGEFEED`]({% link v23.1/create-changefeed.md %})
+    - [`CREATE SCHEDULE FOR CHANGEFEED`]({% link v23.1/create-schedule-for-changefeed.md %})
+    - [`ALTER CHANGEFEED SET {sink}`]({% link v23.1/alter-changefeed.md %})
+    - [`IMPORT`]({% link v23.1/import.md %})
+    - [`EXPORT`]({% link v23.1/export.md %})
+    - [`CREATE EXTERNAL CONNECTION`]({% link v23.1/create-external-connection.md %})
+    - [`COPY`]({% link v23.1/copy-from.md %})
+
+    [#127537][#127537]
+- The cluster setting [`server.jwt_authentication.issuers`]({% link v23.1/sso-sql.md %}) can now take multiple values to support various kinds of providers and their mapped JWKS URI. This can be set to one of the following values:
```

Comment:
it would be a little nicer IMO to link the cluster setting to the 'cluster settings' section of that page: https://www.cockroachlabs.com/docs/v23.1/sso-sql#cluster-settings